### DISCUSSION TITLE
centos7.9 How to choose

### DISCUSSION THREAD

**Discussion by NickCc6** [rabbitmq/rabbitmq-server#8438] [Questions] [answered]
my version is CentOS Linux release 7.9.2009 (Core)

I am not very familiar with the issues with Linux, and I would like to know which download I should choose in the latest 3.11.17

I learned from this that this version is no longer supported? #7527 
If so, can there be any other solution.

Thank you

**Comment by NickCc6** [comment, marked as answer]
I have completed the installation by

rabbitmq-server-generic-unix-3.11.17.tar.xz

**Reply by waitspring** [reply]
Hey, NickCc6:

I find deploy `RabbitMQ 3.11.17` need `Erlang 25.0` with  `OpenSSL 1.1.*` lib file, but `Erlang 23.3.4.18` is latest version in `CentOS7`/ `RHEL7` with `OpenSSL 1.0.*` lib file.
Could you try to deploy `RabbitMQ 3.11.17` in `CentOS7`/ `RHEL7` with `Erlang 25.0` successfully?

version list: https://www.rabbitmq.com/docs/3.13/which-erlang#compatibility-matrix

<img width="938" height="241" alt="图片" src="https://github.com/user-attachments/assets/6c989e0a-dd1c-4755-b9de-52baeb2deeb0" />

**Comment by michaelklishin** [comment]
CentOS 7 is [no longer supported](https://blog.rabbitmq.com/posts/2022/04/centos-7-support-discontinued/). 

RabbitMQ does not have any OS-specific code. As long as you have a suitable version of Erlang, [which our team occasionally produces](https://github.com/rabbitmq/erlang-rpm/) (there are no guarantees that this will continue, none of the customers under Red Hat's Super Extended Very Extended Really Extended 10 Year Support™ pay us anything), you should be able to run the generic binary build. Probably not the RPM package because it has a few more dependencies that CentOS 7 won't satisfy.

But most people should consider CentOS 7 a dead distribution on which their RabbitMQ upgrade options will cease to exist quite soon. And we won't investigate any CentOS 7-specific issues.

**Reply by NickCc6** [reply]
Thank you very much for your answer
